app/controllers/participant_groups_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ParticipantGroupsController < ApplicationController
+
+  load_and_authorize_resource
+
+  # PATCH/PUT /participant_group/1/group
+  def update
+    if @participant_group.update_attributes(group_param)
+      redirect_to :back, notice: I18n.t('participant_groups.update.successful') rescue ActionController::RedirectBackError redirect_to root_path
+    else
+      redirect_to :back, alert: I18n.t('participant_groups.update.failed') rescue ActionController::RedirectBackError redirect_to root_path
+    end
+  end
+
+  private
+
+    def group_param
```

Comment:
Are you sure? I thought participant_group_params should be plural, since it would be for all/several parameters of ParticipantGroup, and group_param would be singular, since it's only for the group parameter.